composer.json

```diff
@@ -15,7 +15,7 @@
     "bin-dir": "bin"
   },
   "require": {
-    "php": ">=5.3.3",
+    "php": ">=5.4",
```

Comment:
```suggestion
    "php": ">=5.6.0",
```

Reply:
@MikeyMJCO I'd be happy with 7.1 - but at least this gets past the immediate issue...

@totten can you merge or merge the number you are happy to go to?

Reply:
So would I ;-)

5.6.0 aligns it with CV's minimum version (which is included in this composer.json)

Reply:
Yeah, I think 5.6 is good for now. (It's rare, but I did recently have to reproduce something where the configuration could only be initialized in PHP 5 + CIvi 4...) IMHO, we should bump this to php7 in a few months.

Also, the `composer.lock` needs  a small twiddling (`composer update --lock`).

Pushed up both those bits.